guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Server.adoc

```diff
@@ -1,6 +1,10 @@
 [[sect-Administering-Renaming_a_Server]]
 == Renaming {ProjectServer} or {SmartProxyServer}
 
+ifndef::satellite[]
+include::snip_katello-only.adoc[]
+endif::[]
```

Comment:
Can't we only include the whole section if Katello is defined?

Reply:
Ditto.